INFO - 2025-01-10 11:07:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-01-10 11:07:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-01-10 11:07:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-01-10 11:08:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-01-10 11:55:14 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'customer_feedba...', 3306, '', 0)
#1 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\sourcecode-api-feedback-resto\app\Models\LoginModel.php(14): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\sourcecode-api-feedback-resto\app\Controllers\LoginController.php(14): App\Models\LoginModel->getDataUsers('', 'e3b0c44298fc1c1...')
#8 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\CodeIgniter.php(933): App\Controllers\LoginController->login_action()
#9 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
#10 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\sourcecode-api-feedback-resto\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\sourcecode-api-feedback-resto\app\Models\LoginModel.php(14): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\sourcecode-api-feedback-resto\app\Controllers\LoginController.php(14): App\Models\LoginModel->getDataUsers('', 'e3b0c44298fc1c1...')
#7 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\CodeIgniter.php(933): App\Controllers\LoginController->login_action()
#8 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
#9 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\sourcecode-api-feedback-resto\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2025-01-10 11:55:14 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: login_action]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
AND `password` = :password:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Models\LoginModel.php(14): CodeIgniter\BaseModel->first()
 6 APPPATH\Controllers\LoginController.php(14): App\Models\LoginModel->getDataUsers('', 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\LoginController->login_action()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\sourcecode-api-feedback-resto\\public\\index.php')
ERROR - 2025-01-10 11:56:43 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'customer_feedba...', 3306, '', 0)
#1 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\sourcecode-api-feedback-resto\app\Models\LoginModel.php(14): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\sourcecode-api-feedback-resto\app\Controllers\LoginController.php(14): App\Models\LoginModel->getDataUsers('hodakova@morzha...', 'e3b0c44298fc1c1...')
#8 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\CodeIgniter.php(933): App\Controllers\LoginController->login_action()
#9 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
#10 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\sourcecode-api-feedback-resto\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\sourcecode-api-feedback-resto\app\Models\LoginModel.php(14): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\sourcecode-api-feedback-resto\app\Controllers\LoginController.php(14): App\Models\LoginModel->getDataUsers('hodakova@morzha...', 'e3b0c44298fc1c1...')
#7 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\CodeIgniter.php(933): App\Controllers\LoginController->login_action()
#8 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
#9 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\sourcecode-api-feedback-resto\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2025-01-10 11:56:43 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: login_action]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
AND `password` = :password:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Models\LoginModel.php(14): CodeIgniter\BaseModel->first()
 6 APPPATH\Controllers\LoginController.php(14): App\Models\LoginModel->getDataUsers('hodakova@morzhan.com', 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\LoginController->login_action()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\sourcecode-api-feedback-resto\\public\\index.php')
ERROR - 2025-01-10 11:57:02 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'customer_feedba...', 3306, '', 0)
#1 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\sourcecode-api-feedback-resto\app\Models\LoginModel.php(14): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\sourcecode-api-feedback-resto\app\Controllers\LoginController.php(14): App\Models\LoginModel->getDataUsers('hodakova@morzha...', '9e279768842d428...')
#8 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\CodeIgniter.php(933): App\Controllers\LoginController->login_action()
#9 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
#10 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\sourcecode-api-feedback-resto\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\sourcecode-api-feedback-resto\app\Models\LoginModel.php(14): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\sourcecode-api-feedback-resto\app\Controllers\LoginController.php(14): App\Models\LoginModel->getDataUsers('hodakova@morzha...', '9e279768842d428...')
#7 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\CodeIgniter.php(933): App\Controllers\LoginController->login_action()
#8 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
#9 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\sourcecode-api-feedback-resto\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\sourcecode-api-feedback-resto\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2025-01-10 11:57:02 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: login_action]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
AND `password` = :password:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Models\LoginModel.php(14): CodeIgniter\BaseModel->first()
 6 APPPATH\Controllers\LoginController.php(14): App\Models\LoginModel->getDataUsers('hodakova@morzhan.com', '9e279768842d4282595f75b1cccc13e96ad4b1b3b683eea3d424438662bf0174')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\LoginController->login_action()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\sourcecode-api-feedback-resto\\public\\index.php')
INFO - 2025-01-10 11:57:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-01-10 11:57:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-01-10 12:03:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-01-10 12:03:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-01-10 12:03:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-01-10 12:03:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-01-10 18:42:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-01-10 18:42:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-01-10 18:42:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-01-10 18:42:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
